aptos-move/e2e-move-tests/src/tests/module_event.rs

```diff
@@ -45,7 +45,7 @@ fn test_module_event_enabled() {
         vec![bcs::to_bytes(&10u64).unwrap()],
     );
     let events = h.get_events();
-    assert_eq!(events.len(), 12);
+    assert_eq!(events.len(), 13);
```

Comment:
This is because we emit `PublishPackage` event in `code.move` when a package gets published.